firmware/config/engines/custom_engine.cpp

```diff
@@ -182,9 +182,9 @@ void setCustomEngineConfiguration(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
 	// todo: 8.2 or 10k?
 	engineConfiguration->vbattDividerCoeff = ((float) (10 + 33)) / 10 * 2;
 
-#if EFI_PROD_CODE
+#if (EFI_PROD_CODE && EFI_CAN_SUPPORT) || defined(__DOXYGEN__)
```

Comment:
I think just EFI_CAN_SUPPORT would be enough here, why would we need both (EFI_PROD_CODE and EFI_CAN_SUPPORT?